### PR TITLE
promed For Pages

### PR DESCRIPTION
**Pull request by aishwaryakapse**
Enter interval of days as cmd line parameter to get the results.

#66 

@JavierJia @MoniMoledo @chenlica @kaiyim

### CODE REVIEW

File: noah/src/main/resources/crawler/promed_mail.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        else:
+            pass         
+
+if __name__ == '__main__':
+    global pageNum
+    pagenum = 0
+    submit = "next"
+    
+    if len(sys.argv) <= 1:
+        sys.exit("Enter the interval in arguments: ")
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]:
```
"Enter the interval in arguments: "
```
to 
```
"Enter the days in arguments: "
```

File: noah/src/main/resources/crawler/promed_mail.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        
+    for arg in sys.argv:
+        interval = arg
+        print interval
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: remove it.

File: noah/src/main/resources/crawler/promed_mail.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+if __name__ == '__main__':
+    global pageNum
+    pagenum = 0
+    submit = "next"
+    
+    if len(sys.argv) <= 1:
+        sys.exit("Enter the interval in arguments: ")
+        
+    for arg in sys.argv:
+        interval = arg
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: `interval = arg[0]` or `interval = arg[1]`

File: noah/src/main/resources/crawler/promed_mail.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    headers['Connection'] = "keep-alive"
+    headers['Content-Type'] = "application/x-www-form-urlencoded"
+    headers['Host'] = 'promedmail.org'
+    
+    end_date = datetime.date.today()
+    start_date = end_date - datetime.timedelta(int(interval))
+    end_date = end_date.strftime("%m/%d/%Y")
+    start_date = start_date.strftime("%m/%d/%Y")
+    filename = "promedsearchresult.html"
+    dirName = "PromedResult"
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: it's better to make it part as a cmd parameter as well.

Comment (aishwaryakapse) [inline noah/src/main/resources/crawler/promed_mail.py]: Done...

File: noah/src/main/resources/crawler/promed_mail.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        else:
+            pass         
+
+if __name__ == '__main__':
+    global pageNum
+    pagenum = 0
+    submit = "next"
+    
+    if len(sys.argv) <= 2:
+        sys.exit("Enter the number of days in arguments: ")
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: add the hint for `dirName`

File: noah/src/main/resources/crawler/promed_mail.py [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        response = urllib2.urlopen(request)
+
+        if not os.path.isfile(completeName):
+            with open(completeName, 'wb') as f:
+                f.write(zlib.decompress(response.read(), 16+zlib.MAX_WBITS))
+        else:
+            pass         
+
+if __name__ == '__main__':
+    global pageNum
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: remove `global`.

File: noah/src/main/resources/crawler/promed_mail.py [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    
+    end_date = datetime.date.today()
+    start_date = end_date - datetime.timedelta(int(interval))
+    end_date = end_date.strftime("%m/%d/%Y")
+    start_date = start_date.strftime("%m/%d/%Y")
+    filename = "promedsearchresult.html"
+       
+    if not os.path.exists(dirName):
+        os.makedirs(dirName)
+        completePath = os.path.abspath(dirName)
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: remove this line.

File: noah/src/main/resources/crawler/promed_mail.py (hunk 6 above)
Comment (aishwaryakapse) [inline noah/src/main/resources/crawler/promed_mail.py]: Done.

File: noah/src/main/resources/crawler/promed_mail.py (hunk 7 above)
Comment (aishwaryakapse) [inline noah/src/main/resources/crawler/promed_mail.py]: done.

File: noah/src/main/resources/crawler/promed_mail.py (hunk 5 above)
Comment (aishwaryakapse) [inline noah/src/main/resources/crawler/promed_mail.py]: Done.

**Comment by JavierJia** [comment]
👍  you can merge it when the test finish.

**Pull request merged**